src/main/java/org/breedinginsight/brapi/v2/services/BrAPIGermplasmService.java

```diff
@@ -138,14 +138,14 @@ public List<Map<String, Object>> processListData(List<BrAPIGermplasm> germplasm,
                 }
             }
 
-            if ((germplasmEntry.getPedigree() != null) && (!germplasmEntry.getPedigree().isEmpty())) {
-                Pedigree germPedigree = Pedigree.parsePedigreeString(germplasmEntry.getPedigree());
```

Comment:
If `Pedigree::parsePedigreeString` is no longer used, can that method be deleted?

Reply:
It could probably could be but since it's in a little utility class maybe we should keep it around in case there's any need for it in the future?

Reply:
Makes sense.